### PR TITLE
Editing help for transfer and transfer_original

### PR DESCRIPTION
**Pull request by ProkhorZ**
1. For clarity, I want to focus the help text for the 'transfer' command on the most typical use case (a single payment).
2. New users will prefer to use 'transfer', so the older method 'transfer_original' should refer to 'transfer' rather than the other way around.

### CODE REVIEW

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -612,8 +612,8 @@ simple_wallet::simple_wallet()
...
-  m_cmd_binder.set_handler("transfer_original", boost::bind(&simple_wallet::transfer, this, _1), tr("transfer [<mixin_count>] <addr_1> <amount_1> [<addr_2> <amount_2> ... <addr_N> <amount_N>] [payment_id] - Transfer <amount_1>,... <amount_N> to <address_1>,... <address_N>, respectively. <mixin_count> is the number of extra inputs to include for untraceability (from 2 to maximum available)"));
-  m_cmd_binder.set_handler("transfer", boost::bind(&simple_wallet::transfer_new, this, _1), tr("Same as transfer_original, but using a new transaction building algorithm"));
+  m_cmd_binder.set_handler("transfer_original", boost::bind(&simple_wallet::transfer, this, _1), tr("Same as transfer, but using an older transaction building algorithm"));
+  m_cmd_binder.set_handler("transfer", boost::bind(&simple_wallet::transfer_new, this, _1), tr("transfer [<mixin_count>] <address> <amount> [<payment_id>]. Pay <amount> to <address>. <mixin_count> is the number of extra inputs to include for untraceability (from 2 to maximum available). Multiple payments can be made at once by adding <address_2> <amount_2> [<payment_id_2>] etcetera at the end"));
```
Comment (moneromooo-monero) [inline src/simplewallet/simplewallet.cpp]: The last sentence is actually wrong. You can use only a single payment id. I'm fine with the rest of the changes. Though minimum mixin will soon go up, so maybe remove the mention of minimum being 2 while you're at it.

**Comment by NanoAkron** [comment]
@ProkhorZ have you seen this comment on your PR?

**Comment by ProkhorZ** [comment]
@NanoAkron Yes, I'm just slow... What do you think about removing the minimum mixin?
Whatever we choose, I made a note to edit this in September again.

**Comment by NanoAkron** [comment]
Just remove the bit in brackets (from 2 to maximum available)

File: src/simplewallet/simplewallet.cpp (hunk 1 above)
Comment (ProkhorZ) [inline src/simplewallet/simplewallet.cpp]: Thanks! Sorry for the error - not understanding this very well was the reason to change it in the first place. But I'll revert to more or less the original text, in order to keep it unambiguous.

**Comment by moneromooo-monero** [comment]
Should be just one commit, please. Also, your original modification (using just one set of address/amount) seems gone, not sure if that was intended.

**Comment by NanoAkron** [comment]
Use `git rebase -i HEAD~2` to rebase the two commits then `pick` the top one and `squash` the next, edit the commit message, then use `push -force` to overwrite the version on your remote repo.

**Comment by NanoAkron** [comment]
@ProkhorZ ping

**Comment by NanoAkron** [comment]
@ProkhorZ ping

**Comment by NanoAkron** [comment]
Anyone mind if I just do this? I'll mention @ProkhorZ in the commit message

**Comment by amiuhle** [comment]
Can't the PR just be squished on GitHub when it's accepted?

**Comment by fluffypony** [comment]
@amiuhle no, not possible - has to be done in the PR, or in a separate PR

**Comment by amiuhle** [comment]
https://github.com/blog/2141-squash-your-commits

@fluffypony Maybe since the repository is quite old, you have to enable this in the settings.

**Comment by fluffypony** [comment]
@amiuhle we don't use the merge button, all merges are GPG signed:) We also want a coherent commit history that lets us follow individual commits, so not everything should be squashed.

**Comment by ProkhorZ** [comment]
@moneromooo-monero Sorry, was editing the branch enough to update the pull request? I removed the original modification because "add stuff at the end" was incorrect. What do you think of the following version? Do we have unlimited space?
`transfer [<mixin_count>] <address> <amount> [<payment_id>] - Transfer <amount> to <address>. <mixin_count> is the number of extra inputs to include for untraceability. Multiple payments can be made at once by adding <address_2> <amount_2> etc. (before the payment ID, if it's included).`

@NanoAkron That's like telling a dog to sign his name. I don't understand git and I don't have access to a Linux terminal. I'm using Github, and I just want to change two lines of text without making a big fuss. Can't I simply delete the second pull request?

**Comment by moneromooo-monero** [comment]
That looks good to me. Making a new PR is probably easiest then if you don't know git. Thanks.

**Comment by NanoAkron** [comment]
@ProkhorZ please submit your updated version. I would much rather see credit go to the original author.

**Comment by ProkhorZ** [comment]
Thanks for your patience. I edited my branch with the version proposed 4 days ago. Can you take it from here?

**Comment by ProkhorZ** [comment]
Do you want me to delete this branch, close this thread and create a completely new pull request?

**Comment by moneromooo-monero** [comment]
Yes, please. Can you point to your branch first, so we can check if there first, in case there are more comments :)

Sorry for the protracted thing.

**Comment by ProkhorZ** [comment]
OK, here's the new one, from a fresh branch: https://github.com/monero-project/monero/pull/1910

**Pull request closed without merging**